tests/conftest.py

```diff
@@ -703,6 +703,46 @@ def callback(request):
     )
 
 
+@pytest.fixture
+def mock_draft_send_unsaved_response(mocked_responses, api_url):
+    body = {
+        "id": None,
+        "bcc": [],
+        "body": "Our latest sale!",
+        "cc": [],
+        "date": 1438684486,
+        "events": [],
+        "files": [],
+        "folder": None,
+        "from": [{"email": "benb@nylas.com"}],
+        "namespace_id": "384uhp3aj8l7rpmv9s2y2rukn",
+        "object": "draft",
+        "reply_to": [],
+        "reply_to_message_id": None,
+        "snippet": "",
+        "starred": False,
+        "subject": "Newsletter",
+        "thread_id": None,
+        "to": [{"email": "my.friend@example.com", "name": "My Friend"}],
+        "unread": False,
+        "version": 0,
+    }
+
+    values = [(400, {}, "Couldn't send email"), (200, {}, json.dumps(body))]
+
+    def callback(request):
+        payload = json.loads(request.body)
+        payload["draft_id"] = "2h111aefv8pzwzfykrn7hercj"
+        return values.pop()
```

Comment:
what happens with the payload? its parsed but isn't used?

Reply:
Great call out. I copied over the mock from another existing mock but I agree it doesn't make too much sense nor does it need to be so complicated. I simplified the mock to fit what is actually needed. For the specific test case I used this for I don't need the payload but for the sake of mocking I still return a payload. I think this SDK needs a refactor of the test cases at some point.